code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -686,3 +686,32 @@
 	var/ramp_up_final = clamp(round(meteorminutes/rampupdelta), 1, 10)
 
 	spawn_meteors(ramp_up_final, wavetype)
+
+//////////////////////////////////////////////
+//                                          //
+//               NATIONS (oh god)           //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/nations
```

Comment:
```suggestion
/// Ruleset for Nations
/datum/dynamic_ruleset/roundstart/nations
```

I hate the ascii shit that ruins doc-comments, need to remove all of them someday

Reply:
based